### PR TITLE
3545 swagger blocks for community care endpoints (supported_sites and eligibility)

### PR DESCRIPTION
**Pull request by sjbrush**
## Description of change
Created ruby with swagger blocks and schemas so that actual swagger can be generated.

## Testing done
None

## Testing planned
Generating swagger?

## Acceptance Criteria (Definition of Done)
Swagger gets generated.

#### Applies to all PRs

- [ ] Appropriate logging
- [X ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by jholton** [comment]
CI is failing with a linting issue.

**Comment by sjbrush** [comment]
codeclimate thinks the two classes are too similar.  they are similar, but they are different classes so i think that should be ignored.  Although, i can't get details why CI failed.  (I get 'no internet' so must not have permissions or something)  so let me ask someone to look.

**Comment by sjbrush** [comment]
ok, so can someone please mark ignore for those codeclimate complaints about similar classes?  Thanks.

**Comment by jholton** [comment]
> ok, so can someone please mark ignore for those codeclimate complaints about similar classes? Thanks.

It's fine.  The Codeclimate checks are more of a suggestion.  You can merge without them passing.

**Comment by sjbrush** [comment]
oh ok, so just waiting on approving review.  sounds good. thanks.

File: modules/vaos/app/swagger/vaos/requests/cc_eligibiility.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          end
+
+          response 403 do
+            key :description, 'Forbidden: user is not authorized for VAOS'
+            schema do
+              key :'$ref', :Errors
+            end
+          end
+
+          response 400 do
```
Comment (jholton) [inline modules/vaos/app/swagger/vaos/requests/cc_eligibiility.rb]: Can you re-order this (move `response 400` block between `200` and `401` blocks)?

**Comment by jholton** [comment]
> oh ok, so just waiting on approving review. sounds good. thanks.

Yeah, the `continuous-integration/jenkins/pr-head` check is required.
The others are useful in identifying some potential issues, but not strictly required for merging.

File: modules/vaos/app/swagger/vaos/requests/cc_eligibiility.rb (hunk 1 above)
Comment (sjbrush) [reply modules/vaos/app/swagger/vaos/requests/cc_eligibiility.rb]: Done!  I'll pay more attention to that in the future.

**Pull request merged**